src/locales/en/messages.json

```diff
@@ -155,9 +155,9 @@
 
   "_manga": "Manga",
 
-  "_jobs": "Jobs",
+  "_jobs": "Tasks ",
```

Comment:
The space here was added intentionally, however it is better not to be added _here_, but **there**:

https://github.com/leoding86/pixiv-omina/blob/525bee3019a43e5fbb520b56c045a24eebe97220/src/renderer/components/Footer.vue#L13

I will correct it.

Reply:
@leoding86, Or perhaps it will look superfluous in Chinese locale, so it is better to left the space here?